tests/src/test/java/org/sonarqube/tests/Category4Suite.java

```diff
@@ -97,7 +98,9 @@
   WsTest.class,
   // quality profiles
   QualityProfilesUiTest.class,
-  LogsTest.class
+  LogsTest.class,
+  // ce
+  WorkerCountTest.class
```

Comment:
It should be defined in Category5Suite, as it's in this category that tests are started there own orchestrator